src/performance_test_fixture.cpp

```diff
@@ -26,7 +26,7 @@ namespace performance_test_fixture
 {
 
 PerformanceTest::PerformanceTest()
-: suppress_memory_tools_logging(true)
+: suppress_memory_tools_logging(true), are_allocation_measurements_active(false)
```

Comment:
Wait, shouldn't this default to `true`?

Reply:
Ah, nevermind, `SetUp` calls `reset_heap_counters`, which sets it. Mea culpa.

Reply:
It's set to true during reset_heap_allocations(), which is called during SetUp and TearDown. Would you rather it be default true at construction?

Reply:
No, this is fine. I just didn't spot it.